sdk/eventhub/Azure.Messaging.EventHubs/Azure.Messaging.EventHubs.sln

```diff
@@ -84,9 +84,6 @@ Global
 		{FB10A6B3-88DD-4296-9241-53ECDF942842}.Release|x64.ActiveCfg = Release|Any CPU
 		{FB10A6B3-88DD-4296-9241-53ECDF942842}.Release|x64.Build.0 = Release|Any CPU
 		{FB10A6B3-88DD-4296-9241-53ECDF942842}.Release|x86.ActiveCfg = Release|Any CPU
-<<<<<<< HEAD
```

Comment:
Looks like a bad merge.

Reply:
It sure does.   Something got cross-threaded yesterday, for sure.